expression/builtin_math.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
-	utilMath "github.com/pingcap/tidb/util/math"
+	utilmath "github.com/pingcap/tidb/util/mathutil"
```

Comment:
Please remove the dependency of github.com/cznic/mathutil, and replace the constants with constants from standard library "math" and Max/Min with the new mathutil package and remove the alias.

Reply:
Done